prismic-model/js/parts/image.js

```diff
@@ -1,27 +1,38 @@
 // @flow
 
-export default function(label: string) {
+type Thumbnails = Array<{
```

Comment:
nit: General we don't type arrays of things, so we would just type `Thumbnail`, and below use Array<Thumbnail>, making `Thumbnail` more accessible.

Reply:
Thanks, i have updated this and place this in model which can be used later to contain types etc..